bundle/edu.gemini.catalog/src/main/java/jsky/catalog/RangeSearchCondition.java

```diff
@@ -113,12 +85,12 @@ public boolean isTrueFor(Comparable val) {
                     && (_maxVal == null || _maxVal.compareTo(val) >= 0));
         }
 
-        if (_minInclusive && !_maxInclusive) {
+        if (_minInclusive) {
             return ((_minVal == null || _minVal.compareTo(val) <= 0)
                     && (_maxVal == null || _maxVal.compareTo(val) > 0));
         }
 
-        if (!_minInclusive && _maxInclusive) {
+        if (_maxInclusive) {
             return ((_minVal == null || _minVal.compareTo(val) < 0)
                     && (_maxVal == null || _maxVal.compareTo(val) >= 0));
         }
```

Comment:
Is this logic change valid? What if it's both `_minInclusive` and `_maxInclusive`?